fs_overlay/var/lib/nginx-conf/default.ssl.conf.erb

```diff
@@ -1,3 +1,11 @@
+<% if domain.multiple_upstreams? %>
+upstream <%= domain.upstream_backend_name %> {
+    <% domain.upstreams.each do |upstream| %>
+        server <%= upstream[:upstream] %> <%= upstream[:parameters] %>;
```

Comment:
Let's call it `upstream[:address]`, shall we? 
We have too many occurrences of the word "upstream" here.
And the [Nginx doc](http://nginx.org/en/docs/http/ngx_http_upstream_module.html#upstream) calls it `address`: 
> Syntax:	server address [parameters];

Reply:
Good point. `address` is a much better name.